src/components/Portfolio/LoanStatsTable.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+	<div class="tw-flex tw-justify-between tw-mb-2 tw-bg-[#efefef] tw-p-2">
+		<div class="tw-flex-1"></div>
+		<div class="tw-flex-1 tw-text-right tw-font-medium">
+			My stats
+		</div>
+		<div class="tw-flex-1 tw-text-right tw-font-medium">
+			Avg Kiva lender
+		</div>
+	</div>
+
+	<div v-if="loading" class="tw-space-y-2">
+		<div v-for="i in 13" :key="i" class="tw-grid tw-grid-cols-12 tw-gap-4">
+			<kv-loading-placeholder
+				class="tw-col-span-4"
+				style="width: 120px; height: 16px;"
+			/>
+			<kv-loading-placeholder
+				class="tw-col-span-4 tw-ml-auto"
+				style="width: 80px; height: 16px;"
+			/>
+			<kv-loading-placeholder
+				class="tw-col-span-4 tw-ml-auto"
+				style="width: 80px; height: 16px;"
+			/>
+		</div>
+	</div>
+
+	<div v-else class="tw-space-y-2">
+		<div
+			v-for="row in statsRows"
+			:key="row.key"
+			class="tw-grid tw-grid-cols-12 tw-gap-4"
+		>
+			<div class="tw-col-span-4">
+				{{ row.label }}
+			</div>
+			<div class="tw-col-span-4 tw-text-right">
+				{{ formatValue(stats[row.key], row.type) }}
+			</div>
+			<div class="tw-col-span-4 tw-text-right">
+				{{ formatValue(avgStats[row.key], row.type) }}
+			</div>
+		</div>
+	</div>
+
+	<hr class="tw-border-tertiary tw-my-4">
+
+	<div class="tw-grid tw-grid-cols-1 md:tw-grid-cols-2 tw-gap-x-2">
```

Comment:
In general this table is not following best practices for accessibility. Is there any way we can make this an actual table? If not, I think we should at least add aria table roles to these elements: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/table_role

Reply:
Agreed for the first section -- I've converted that to a table.
The second section however wants to split into multiple columns at larger screen sizes, and I'm not sure tables support this kind of behavior the way grids do...  So I've opted for setting an aria role for the second section to address this.  Any thoughts on that?